app/controllers/admin/order_cycles_controller.rb

```diff
@@ -70,6 +69,12 @@ def clone
       redirect_to main_app.admin_order_cycles_path, :notice => "Your order cycle #{@order_cycle.name} has been cloned."
     end
 
+    # Send notifications to all producers who are part of the order cycle
+    def notifications
```

Comment:
I would expect an action named "notifications" to display a list of notifications. However, what we're doing here is an action, so I would use a verb to name it, probably `notify_producers`.